data/src/main/java/io/gnosis/data/models/transaction/TransactionDetails.kt

```diff
@@ -37,7 +39,11 @@ data class TxData(
     @Json(name = "value")
     val value: BigInteger?,
     @Json(name = "operation")
-    val operation: Operation
+    val operation: Operation,
+
+    //FIXME: suboptimal backend response structure
```

Comment:
This will be different with the new endpoint, right?

Reply:
I hope so